nal_eval/tests/fixtures/while_loop.nal

```diff
@@ -0,0 +1,11 @@
+let mut x = 0
+
+while x < 5 {
+    print(x)
+    x = x + 2
+}
+
+while x > 0 {
+    print(x)
+    x = x - 3
+}
```

Comment:
This is also a valid test. Thanks!
I found a silly mistake that makes this test failing. I'll fix it in next commit.